schema.sql

```diff
@@ -23,7 +23,7 @@ CREATE TABLE owners (
 
 CREATE TABLE species (
     id serial PRIMARY KEY,
-    name VARCHAR(255)
+    specie_name VARCHAR(255)
```

Comment:
:orange_book: [OPTIONAL] Please Don't add unnecessary context to column names. Column names should be descriptive, but they should also be concise. Avoid adding unnecessary context to column names, such as the table name. For example, instead of `animal_id`, `animal_name` you could simply use `id`, `name` when referring to the animals table's columns